cluster/examples/kubernetes/ceph/cluster-stretched.yaml

```diff
@@ -68,3 +68,5 @@ spec:
               values:
               - b
               - c
+  disruptionManagement:
+    managePodBudgets: false
```

Comment:
Is there a reason this has to be `false` for stretch clusters that we might want to document as a comment?

Reply:
copy-paste issue, I intended to set them all to true